### PR TITLE
feat: event list view

### PR DESCRIPTION
**Pull request by karenzhao35**
Sample for ticket 
image

Events-List populated using dummy event data in place of results from a database query as a proof of concept

### CODE REVIEW

**Review by rustamch — APPROVED** [review]
As a first round for the further iteration, it looks good to me!

File: app/events/event.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+
+import React from 'react';
+
+interface EventProps {
```
Comment (steph-feng) [inline app/events/event.tsx:5]: should we add the date to the interface?

Comment (janayee-c) [reply app/events/event.tsx:5]: ^ to clarify, is the props.volunteers showing the number of volunteers in total for the event or the number of volunteers still needed?

Comment (karenzhao35) [reply app/events/event.tsx:5]: It was meant to just be the total number of volunteers needed (but I guess we could also make it decrement as more volunteers sign up?)

Comment (janayee-c) [reply app/events/event.tsx:5]: Okay thank you! Sounds good just thinking about future iterations this looks solid to me.

**Review by janayee-c — APPROVED** [review]

**Pull request merged**